templates/common/baremetal/files/baremetal-keepalived.yaml

```diff
@@ -23,31 +23,8 @@ contents:
       - name: conf-dir
         hostPath:
           path: "/etc/keepalived"
-      initContainers:
-      - name: render-config
-        image: {{ .Images.baremetalRuntimeCfgImage }}
-        command:
-        - runtimecfg
```

Comment:
Is runtimecfg still used by anyone or it can be dropped from https://github.com/openshift/baremetal-runtimecfg/?

Reply:
It is used by mdns-publisher